src/main/resources/db/views/R__101_serious_sexual_summary.sql

```diff
@@ -0,0 +1,70 @@
+drop materialized view if exists serious_sexual_assault_summary;
+
+create materialized view serious_sexual_assault_summary as
```

Comment:
it looks like there will be quite a few of these: i think it'd be nice to prefix the views with some kind of DPR-related prefix so we can tell what they're for. what do you think?

Reply:
I'll do that in the next pr